docs/guide/tutorial-8-using-ros.md

```diff
@@ -14,23 +14,35 @@ To install the latest version of ROS on Ubuntu use the following commands:
 sudo sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main" > /etc/apt/sources.list.d/ros-latest.list'
 sudo apt-key adv --keyserver 'hkp://keyserver.ubuntu.com:80' --recv-key C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654
 sudo apt-get update
-sudo apt-get install ros-melodic-desktop-full
+sudo apt-get install ros-melodic-desktop-full # takes time, get a coffee :)
 sudo rosdep init
 rosdep update
+sudo apt-get install ros-melodic-webots-ros
 ```
 
 For more information or to install it on another platform please read [http://wiki.ros.org/ROS/Installation](http://wiki.ros.org/ROS/Installation).
 Unless you need older version for some other application, you should choose the latest distribution (Melodic Morenia or Kinetic Kame).
 
-If you never used the ROS framework before, it is strongly recommended to follow some tutorials from: [http://wiki.ros.org/ROS/Tutorials](http://wiki.ros.org/ROS/Tutorials).
+The last line is to install the [webots\_ros](http://wiki.ros.org/webots\_ros) package.
+
+If you never used the ROS framework before, it is strongly recommended to follow some tutorials (_Beginner Level_) from: [http://wiki.ros.org/ROS/Tutorials](http://wiki.ros.org/ROS/Tutorials).
 These tutorials will also help you set up your ROS environment and initialize your catkin workspace.
+The minimum requirement is to follow these instructions (taken from the [ROS tutorial 1.1.1](http://wiki.ros.org/ROS/Tutorials/InstallingandConfiguringROSEnvironment)).
+1. Open a terminal
+2. Check if the environment is source : `printenv | grep ROS`
```

Comment:
"Check if the environment is source :" &rarr; "Check if the ROS environment is properly set-up:"